client/post-editor/editor-revisions-list/header.jsx

```diff
@@ -7,15 +7,48 @@
 import React from 'react';
 import PropTypes from 'prop-types';
 import { localize } from 'i18n-calypso';
+import Gridicon from 'gridicons';
 
-const EditorRevisionsListHeader = ( { numRevisions, translate } ) => {
+/**
+ * Internal dependencies
+ */
+import Button from 'components/button';
+
+const EditorRevisionsListHeader = ( {
+	numRevisions,
+	translate,
+	selectNextRevision,
+	selectPreviousRevision,
+} ) => {
 	return (
-		<div className="editor-revisions-list__header">
-			{ !! numRevisions &&
-				translate( '%(revisions)d revision', '%(revisions)d revisions', {
-					count: numRevisions,
-					args: { revisions: numRevisions },
-				} ) }
+		<div>
+			<div className="editor-revisions-list__header">
+				{ !! numRevisions &&
+					translate( '%(revisions)d revision', '%(revisions)d revisions', {
+						count: numRevisions,
+						args: { revisions: numRevisions },
+					} ) }
+			</div>
+			<div className="editor-revisions-list__navigation">
+				<Button
+					compact
+					borderless
+					className="editor-revisions-list__prev-button"
+					type="button"
+					onClick={ selectPreviousRevision }
+				>
```

Comment:
We need to set `disabled` on these buttons when we're at the top / bottom of the list respectively.

Reply:
I've committed this improvement in 4578a846503111b058fd01538add35d0db44f9fa.